packages/rust/server/src/routes/user.rs

```diff
@@ -190,62 +205,81 @@ pub struct Email {
 
 async fn forgot(
     db: Extension<DbPool>,
+    email_service: Extension<EmailService>,
     Json(payload): Json<Email>,
 ) -> Result<StatusCode, StatusCode> {
     let user = User::from_email(&db.0, &payload.email)
         .await
-        .map_err(map_sqlx_err)?;
+        .map_err(|_| StatusCode::CREATED)?;
```

Comment:
Why is this StatusCode::CREATED

Reply:
Security reasons to prevent user enumeration. See here: https://cheatsheetseries.owasp.org/cheatsheets/Forgot_Password_Cheat_Sheet.html

Reply:
Technically, usernames/emails can be enumerated through the register API. But we can deal with that later.